mlflow/tracking/_tracking_service/client.py

```diff
@@ -873,6 +873,25 @@ def download_artifacts(self, run_id, path, dst_path=None):
         """
         return self._get_artifact_repo(run_id).download_artifacts(path, dst_path)
 
+    def _log_url(self, run_id):
+        try:
+            host_url = self.store.get_host_creds().host
+        except AttributeError:
+            # If the store does not have host creds, we cannot log the URL. This is expected for
+            # SqlAlchemyStore.
+            return
+        run_info = self.store.get_run(run_id).info
+        experiment_id = run_info.experiment_id
+        run_name = run_info.run_name
+        if self.tracking_uri == "databricks":
+            experment_url = f"{host_url}/ml/experiments/{experiment_id}"
+        else:
+            experment_url = f"{host_url}/#/experiments/{experiment_id}"
+        run_url = f"{experment_url}/runs/{run_id}"
+
+        _logger.info(f"🎊 View run {run_name} at: {run_url}.")
+        _logger.info(f"💫 View your experiment at: {experment_url}.")
```

Comment:
```suggestion
        _logger.info(f"View run {run_name} at: {run_url}.")
        _logger.info(f"View your experiment at: {experment_url}.")
```

Do we need emojis? They don't render well on Windows powershell.

Reply:
I think wandb used emoji to make these 2 lines' logging more visible, I am happy to keep or remove. Windows is the environment I care the least about XD

Reply:
Got it. Let's keep them. Can we use 🧪 for experiment?

Reply:
sure thing!